### PR TITLE
Made some changes in the navbar link and corrected grammar errors

### PR DESCRIPTION
**Pull request by LikkleEgret**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -110,7 +110,7 @@ <h1 class="font-weight-bold mb-4 text-white hero-8-title">
                 Tech Talk 7.0
               </h1>
               <p class="mb-5 text-white-70 w-lg-75 mx-auto">
-                Secrets on how to get into the BIG 5 companies as a developer.
+                <span>Secrets on how to get into the BIG 5 companies as a developer.</span>
```
Comment (timokonkwo) [inline index.html:113]: Please why did you add a span to this?

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+  "liveServer.settings.port": 5501
```
Comment (timokonkwo) [inline .vscode/settings.json:2]: This shouldn't be part of the PR. 

Add `.vscode` in the `.gitignore` file and create another PR to solve this

File: index.html [hunk 3]
```diff
@@ -77,7 +77,7 @@
               <a href="#team" class="nav-link">Speakers</a>
             </li>
             <li class="nav-item">
-              <a href="#gallary" class="nav-link">Gallary</a>
+              <a href="#gallary" class="nav-link">Gallery</a>
```
Comment (timokonkwo) [inline index.html:80]: This has already been solved. When did you fork the repo?

Please go to your fork of the project and click on `Sync` so that your fork will be up to date with the project.

**Review by timokonkwo — COMMENTED** [review]
Your changes are great.

Please refer to the comments in the files and effect the changes in another PR.

Thanks for your contributions

File: index.html (hunk 1 above)
Comment (LikkleEgret) [reply index.html:113]: Wanted to make the sentence a bit bolder in the css file,  it was a bit difficult to see considering it's the first sentence you read once you visit the site

File: index.html (hunk 3 above)
Comment (LikkleEgret) [reply index.html:80]: Three/four days ago

Comment (LikkleEgret) [reply index.html:80]: Ok

File: .vscode/settings.json (hunk 2 above)
Comment (LikkleEgret) [reply .vscode/settings.json:2]: Sure

File: index.html (hunk 1 above)
Comment (timokonkwo) [reply index.html:113]: That's great. Please effect the changes and create another PR

Comment (LikkleEgret) [reply index.html:113]: Kk

Comment (LikkleEgret) [reply index.html:113]: OK.
